# HG changeset patch
# User Rik <rik@octave.org>
# Date 1521653252 25200
#      Wed Mar 21 10:27:32 2018 -0700
# Branch stable
# Node ID 23b5c067805165f9d2a75e812d7874330fb9cbb2
# Parent  f0a427d09dbacfc5b572f6a70e79dfba8dd8337a
mk_qthelp.pl: Change absolute filenames to basesname (filenames) for Qt4 (bug #53399).

* mk_qthelp.pl: Change absolute filenames to basesname (filenames) for
.qhcp file.

diff --git a/doc/interpreter/mk_qthelp.pl b/doc/interpreter/mk_qthelp.pl
--- a/doc/interpreter/mk_qthelp.pl
+++ b/doc/interpreter/mk_qthelp.pl
@@ -167,18 +167,18 @@ print $FH <<__EOT3__;
 __EOT3__
 
 close ($FH);
 
 ################################################################################
 # Prepare Qt Help Collection Project document (.qhcp file)
 open ($FH, ">", "$fname.qhcp") or die "Unable to open $fname.qhcp";
 
-$qhpfile = "$fname.qhp";
-$qchfile = "$fname.qch";
+$qhpfile = basename ($fname) . ".qhp";
+$qchfile = basename ($fname) . ".qch";
 
 # This is the entire file
 print $FH <<__EOT4__;
 <?xml version="1.0" encoding="UTF-8"?>
 <!--DO NOT EDIT! Generated automatically by $0-->
 <QHelpCollectionProject version="1.0">
   <docFiles>
     <generate>
